src/javascripts/modules/SidebarToggler.js

```diff
@@ -38,7 +38,9 @@ PurpleMine.SidebarToggler = (function () {
 
     // Fix issue with context menu position
     if (this.$main.css('position') === 'relative') {
-      $('#context-menu').appendTo('#wrapper3')
+      $(window).load(function () {
+        $('#context-menu').appendTo('#wrapper3')
+      })
```

Comment:
Were there issues with loading this? All JS plugins are loaded on .ready in [src/javascripts/theme.js](/mrliptontea/PurpleMine2/blob/399b5abeb9f2e6a73cce70aec1f6796b59d66da1/src/javascripts/theme.js), so it shouldn't be necessary. 🤔